### PR TITLE
Handle loading models with no device_model_description present

### PR DESCRIPTION
**Pull request by peternewman**
As well as some other fields being empty/missing. Generally try and update the website to match the RDM spec in terms of behaviour.

### CODE REVIEW

File: model.py [hunk 1]
```diff
@@ -59,11 +59,11 @@ class Responder(db.Model):
   # The Device Model ID field from DEVICE_INFO
   device_model_id = db.IntegerProperty()
   # The DEVICE_MODEL_DESCRIPTION
-  model_description = db.StringProperty(required=True)
+  model_description = db.StringProperty(default='')
```
Comment (peternewman) [inline model.py]: I've changed from required, to default='' as per https://github.com/OpenLightingProject/rdm-app/pull/50 , but I'm wondering, why don't we default to None instead, so we can filter/search on responders with no model description?

Comment (nomis52) [reply model.py]: Makes sense.

Comment (peternewman) [reply model.py]: Okay, I've started moving to None and the related kick off in the templates.

Any thoughts on what name a device with no model description should show in http://rdm.openlighting.org/model/browse ? Currently it's just the RDM ID and some words (I wonder if it should include the Manufacturer ID too).

Comment (nomis52) [reply model.py]: That sounds reasonable.

**Comment by peternewman** [comment]
I think this is all sorted now, so would you mind a re-review when you get a chance @nomis52 ?

**Comment by peternewman** [comment]
Thanks, would you mind deploying too when you get a chance @nomis52 .

**Pull request merged**